client/verta/docs/examples/tutorials.rst

```diff
@@ -1,12 +1,15 @@
 Tutorials
 =========
 
-Check out these tutorials for detailed step-by-step instructions on performing different MLOps 
+Check out these tutorials for detailed step-by-step instructions on performing different MLOps
 functions in Verta.
 
 ModelDB (Versioning and Metadata)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+* `Versioning model ingredients <tutorials/ingredients.html>`_, which allows you to have Git-like
+  for the components of models.
+
 * `Versioning a Model <tutorials/versioning.html>`_
 * `Versioning Data <tutorials/dataset.html>`_
 * `Logging and Querying Metadata <tutorials/workflow.html>`_
```

Comment:
It makes sense conceptually to separate "Versioning Model Components" and "Versioning a Model"
but Manasi is saying that the line
```
* `Versioning a Model <tutorials/versioning.html>`_
```
is currently a dead link because it _was meant_ to be a placeholder for the page that this PR is adding.
I don't think it makes sense to keep it if it's no longer a placeholder and also doesn't go anywhere. Plus, the
> versioning the model (via run and logging things)

is covered two bullet points below:
```
* `Logging and Querying Metadata <tutorials/workflow.html>`_
```

To resolve this, I'm suggesting this change:
```suggestion
* `Versioning Model Ingredients <tutorials/ingredients.html>`_, which allows you to have Git-like
  tracking for the components of models
* `Versioning a Model and Logging Metadata <tutorials/workflow.html>`_
* `Versioning Data <tutorials/dataset.html>`_
```

Reply:
Sure. There will be more shifting on the tutorials anyway.